.circleci/config.yml

```diff
@@ -0,0 +1,84 @@
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/Financial-Times/draft-annotations-api
+    docker:
+      - image: golang:1.8.5
+        environment:
+          GOPATH: /go
+          CIRCLE_TEST_REPORTS: /tmp/test-results
+          CIRCLE_COVERAGE_REPORT: /tmp/coverage-results
+    steps:
+      - checkout
+      - run:
+          name: External Dependencies
+          command: |
+            go get -u github.com/mattn/goveralls
+            go get -u github.com/jstemmer/go-junit-report
+            go get -u github.com/kardianos/govendor
+            go get -u github.com/haya14busa/goverage
+      - run:
+          name: Test Results
+          command: |
+            mkdir -p ${CIRCLE_TEST_REPORTS}
+            mkdir -p ${CIRCLE_COVERAGE_REPORT}
+      - run:
+          name: Govendor Sync
+          command: govendor sync -v
+      - run:
+          name: Go Build
+          command: go build -v
+      - run:
+          name: Run Tests
+          command: |
+            govendor test -race -v +local | /go/bin/go-junit-report > ${CIRCLE_TEST_REPORTS}/main.xml
+            goverage -covermode=atomic -race -coverprofile=${CIRCLE_COVERAGE_REPORT}/coverage.out ./...
+      - run:
+          name: Upload Coverage
+          command: /go/bin/goveralls -coverprofile=${CIRCLE_COVERAGE_REPORT}/coverage.out -service=circle-ci -repotoken=$COVERALLS_TOKEN
+      - store_test_results:
+          path: /tmp/test-results
+  dockerfile:
+    working_directory: /draft-content-api
+    docker:
+      - image: docker:18.01.0-ce-git
+    steps:
+      - checkout
+      - setup_docker_engine
+      - run:
+          name: Build Dockerfile
+          command: docker build .
+  dredd:
+    working_directory: /go/src/github.com/Financial-Times/draft-annotations-api
+    docker:
+      - image: bankrs/golang-dredd:go1.8.3-dredd4.4.0
+        environment:
+          GOPATH: /go
+          ANNOTATIONS_RW_ENDPOINT: http://localhost:9000
+          ANNOTATIONS_ENDPOINT: http://localhost:9000/content/%v/annotations
+          INTERNAL_CONCORDANCES_ENDPOINT: http://localhost:9000/internalconcordances
+      - image: peteclarkft/ersatz:stable
```

Comment:
Out of curiosity... where is the Dockerfile for this image?

Reply:
Dockerhub and here: https://github.com/peteclark-ft/ersatz